scheduler/kubernetes/snaps.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: page
-title: "Create and use snapshots"
+title: "Portworx snapshots"
```

Comment:
I think this was correct before. Using Portworx in the heading is redundant. Also, title should ideally match the link  on the sidebar